场景物品说明的翻译补完

Menschheit opened the pull request:
把所有的场景物品说明翻译了一遍（这样就至少不会新游戏一进神庙就看到英文的pot了）。
由于语境问题（有的场景我以前好像没去过，没什么映像），不能保证准确性。

Menschheit committed:
场景物品说明的翻译补完

Menschheit committed:
修正一处严重错误

有个地方为了表达cross and circle不小心用了半角字符⊗，转码时出了问题（被转成了{。。。}的形式），会导致msg文件格式出错，游戏闪退。而且事实上文本中也不应该允许出现ascii码大于128的半角字符，否则会出现无法识别是否为汉字的情况，因为汉字编码第一字节的范围是81h到FEh。
本来想考虑用全角字符代替，后来觉得显示效果也不一定好，于是改回了文字说明。

Menschheit committed:
测试用的新f2_res.dll

调整了每行显示的最大文字量。
初步增加英文的换行机制（数字与单词还有不会被拆开到下一行）
还没测试！
心血来潮加了几行指令，百度网盘上传速度0kB/s，于是我索性传到github上了，反正除了你也没人用这东西（一年前我在网上找乱码问题的解决方案，看到bn13上有个人提出同样的问题，然而并没有人解决。我过了一周从零开始的汇编生活，改了DLL，还回复了那个帖子，结果楼主根本没鸟我，好伤心。。。）

tytannial commented:
好吧。。无法直视的强大。居然更新高分辨率补丁了，要不是自己能力不够，想自己操起C++弄个DLL插件挂上去解决问题。。。汇编这玩意实在太伤了，完全不知从何入手，特别是对我这种写C#这类语言的。。。

你检查下吧，看看编码什么的有没有问题，如果没问题我就合并了。

Menschheit committed:
修正一个可能存在的问题

修正了当单词过长时造成程序崩溃的问题（现在遇到长度超过一行最大显示数量的单词会被强制拆开）

已知问题：突然发现坑爹的辐射2用的字体太古老了(似乎是封装在master.dat里的aaf字体)，显示不同字母，半角符号时所用的宽度都不一样，当一行中出现大量较宽的字符会溢出讯息栏范围，当出现大量较窄的字符时会留有较大空位。
经个人测试原始高分辨率补丁以及没打高分辨率补丁的原版一样有这个问题，你可以试试把某个文本（我通常用试炼神庙的arcaves.msg）改成大量的某种相同字符（比如%或者f），会非常明显。如果汉化比较完整的话由于全角字符用的字体宽度固定所以问题不是很明显。
理想中的解决办法有2个，一是分别测试每个每个字符的宽度然后分类（吃力不讨好本来问题就不大），二是换字体（完全不会- -）

Menschheit commented:
编码应该没问题了，第一次直接在网页上改的，结果自动给我转成unicode了，我现在都是在电脑上转码转好了在上传整个文件。

tytannial commented:
恩好的，只要是GB2312就可以。

tytannial commented:
这两天我有空的话测试下，如果没什么特别大的问题我就发布更新包吧。
